sbtStudentCommands/IC.scala.template

```diff
@@ -0,0 +1,29 @@
+/**
+  * Copyright © 2014, 2015 Typesafe, Inc. All rights reserved. [http://www.typesafe.com]
+  */
+package sbtstudent
+
+import sbt._
+import Keys._
+import sbt.complete.DefaultParsers._
+
+import scala.Console
+import StudentKeys._
+import Zip.withZipFile
+import sbtstudent.Pssr.{getCueFolder, getRootFolder}
+
+object IC {
+  def ICselectFirstExercise: Command = Command.command("ICselectFirstExercise") { state =>
+
+    val firstExercise = state.get(mapNext).get.keys.toVector.sorted.head
+    withZipFile(state, firstExercise) { () =>
+      val currentExerciseMain = new sbt.File(getRootFolder(state), "exercises")
+      val cueFolder = getCueFolder(state)
+      val solFolder = new sbt.File(cueFolder, firstExercise)
+      IO.delete(currentExerciseMain)
+      IO.copyDirectory(solFolder, currentExerciseMain)
```

Comment:
You do have a point about the fact that the solution shouldn't be pulled (as validateStudentRepo.sh does this just before running `test`